.circleci/config.yml

```diff
@@ -160,45 +160,56 @@ workflows:
   build:
     jobs:
       - test:
-          jdk: '8'
+          jdk: 'circleci/openjdk:8'
           name: 'jdk 8'
           filters:
             <<: *all-branches
       - test:
-          jdk: '11'
+          jdk: 'circleci/openjdk:11'
           name: 'jdk 11'
           filters:
             <<: *all-branches
       - test:
-          jdk: '14-buster'
+          jdk: 'circleci/openjdk:14-buster'
           name: 'jdk 14'
```

Comment:
I think as we add jdk 16 "support" we could drop 14.

Reply:
I have an uncommitted change for this, moving to `cimg/openjdk:16.0.0` (since that's where circleci appears to be migrating its images to).